src/explore-education-statistics-frontend/src/modules/subscriptions/ConfirmUnsubscriptionPage.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import { GetServerSideProps, NextPage } from 'next';
+import Button from '@common/components/Button';
+import publicationService, {
+  PublicationTitle,
+} from '@common/services/publicationService';
+import { Dictionary } from '@common/types';
+import Page from '@frontend/components/Page';
+import notificationService, {
+  Subscription,
+} from '@frontend/services/notificationService';
+import withAxiosHandler from '@frontend/middleware/ssr/withAxiosHandler';
+import SubscriptionStatusMessage from '@frontend/modules/subscriptions/components/SubscriptionStatusMessage';
+import Head from 'next/head';
+
+interface Props {
+  publicationSlug: string;
+  data: PublicationTitle;
+  token: string;
+}
+
+const ConfirmUnsubscriptionPage: NextPage<Props> = ({
+  data,
+  publicationSlug,
+  token,
+}) => {
+  const [unsubscribedSubscription, setUnsubscribedSubscription] = useState<
+    Subscription | undefined
+  >(undefined);
+
+  const onConfirmClicked = async () => {
+    // TODO: Currently we get a 400 if the token isn't valid, but aren't
+    // properly handling this like we are for verification errors.
```

Comment:
Do these 400s return an error similar to [EES-3192](https://dfedigital.atlassian.net/browse/EES-3192)? Does there need to be a ticket to implement similar error handling?